Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs

```diff
@@ -42,28 +39,16 @@ class OptionsTreeView : TreeView {
 
         private void InitializeImageList() {
             if (_imageList == null) {
-                using (var imgHandler = new ImageHandler(typeof(OptionsTreeView).Assembly.GetManifestResourceStream("Microsoft.PythonTools.Project.Resources.imagelis.bmp"))) {
-#pragma warning disable 0618
-                    var openFolder = imgHandler.ImageList.Images[(int)ProjectNode.ImageName.OpenFolder];
-                    var closedFolder = imgHandler.ImageList.Images[(int)ProjectNode.ImageName.Folder];
-#pragma warning restore 0618
-
-                    _imageList = new ImageList();
-
-                    Bitmap bmp = new Bitmap(16, 16);
-
-                    // transparent image is the image we use for owner drawn icons
-                    _imageList.TransparentColor = Color.Magenta;
-                    using (var g = Graphics.FromImage(bmp)) {
-                        g.FillRectangle(
-                            Brushes.Magenta,
-                            new Rectangle(0, 0, 16, 16)
-                        );
-                    }
-                    _imageList.Images.Add(bmp);
-                    _imageList.Images.Add(openFolder);
-                    _imageList.Images.Add(closedFolder);
+                _imageList = new ImageList();

```

Comment:
I think we need the empty image to get proper layout - don't really want to dig deep enough to figure out how to get that layout without it, and frankly I'd rather just use a WPF treeview there so we don't have to do the ownerdrawn stuff at all. But that's for a future release.

Right now, _imageList is static and that's how we avoid creating lots of image lists over the life of the process.